deploy/local_single_user_deploy.sh

```diff
@@ -0,0 +1,218 @@
+        .chrono_grapher.Logging.log.file = ($bin_dir + "/" + .chrono_grapher.Logging.log.file) |
+        .chrono_client.Logging.log.file = ($bin_dir + "/" + .chrono_client.Logging.log.file)' "$default_conf" > "$output_file"
+
+    local output_file="${conf_dir}/client_conf.json"
+    jq --arg bin_dir "$bin_dir" \
+       '.chrono_visor.Logging.log.file = ($bin_dir + "/" + .chrono_visor.Logging.log.file) |
+        .chrono_grapher.Logging.log.file = ($bin_dir + "/" + .chrono_grapher.Logging.log.file) |
+        .chrono_client.Logging.log.file = ($bin_dir + "/" + .chrono_client.Logging.log.file)' "$default_conf" > "$output_file"
+
+}
+
+extract_shared_libraries() {
+    local executable="$1"
+    ldd_output=$(ldd ${executable} 2>/dev/null | grep '=>' | awk '{print $3}' | grep -v 'not' | grep -v '^/lib')
+    echo "${ldd_output}"
+}
+
+copy_shared_libs_recursive() {
+    local lib_path="$1"
+    local dest_path="$2"
+    local linked_to_lib_path="$(readlink -f ${lib_path})"
+
+    # Copy the library and maintain symbolic links recursively
+    final_dest_lib_copies=false
+    echo -e "${DEBUG}Copying ${lib_path} recursively ...${NC}"
+    while [ "$final_dest_lib_copies" != true ]
+    do
+        cp -P "$lib_path" "$dest_path/"
+        if [ "$lib_path" == "$linked_to_lib_path" ]
+        then
+            final_dest_lib_copies=true
+        fi
+        lib_path="$linked_to_lib_path"
+        linked_to_lib_path="$(readlink -f ${lib_path})"
+    done
+}
+
+copy_shared_libs() {
+    libs_visor=$(extract_shared_libraries ${VISOR_BIN})
+    libs_keeper=$(extract_shared_libraries ${KEEPER_BIN})
+    libs_client=$(extract_shared_libraries ${CLIENT_BIN})
+
+    # Combine shared libraries from all executables and remove duplicates
+    all_shared_libs=$(echo -e "${libs_visor}\n${libs_keeper}\n${libs_client}" | sort | uniq)
+
+    # Copy shared libraries to the lib directory
+    echo -e "${DEBUG}Copying shared library ...${NC}"
+    mkdir -p ${LIB_DIR}
+    for lib in ${all_shared_libs}; do
+        if [[ ! -z ${lib} ]]
+        then
+            copy_shared_libs_recursive ${lib} ${LIB_DIR}
+        fi
+    done
+}
+
+# Functions to launch and kill processes
+launch_process() {
+    local bin="$1"
+    local args="$2"
+    local log_file="$3"
+    echo -e "${DEBUG}Launching $bin ...${NC}"
+    LD_LIBRARY_PATH=${LIB_DIR} nohup ${bin} ${args} > ${BIN_DIR}${log_file} 2>&1 &
+}
+
+kill_process() {
+    local bin="$1"
+    echo -e "${DEBUG}Killing $(basename ${bin}) ...${NC}"
+    pkill -9 -f ${bin}
+}
+
+stop_process() {
+    local bin="$1"
+    echo -e "${DEBUG}Stopping $(basename ${bin}) ...${NC}"
+    pkill -f ${bin}
+}
+
+# Main functions for install, reset, and usage
+install() {
+    echo -e "${INFO}Installing ...${NC}"
+    copy_shared_libs
+    check_files
+    generate_config_files ${NUM_KEEPERS} ${BIN_DIR} ${CONF_FILE} ${CONF_DIR}
+    launch_process ${VISOR_BIN} "--config ${CONF_DIR}/visor_conf.json" "/visor.log"
```

Comment:
I think it makes more sense to add `/` in `launch_process` than passing `/xxxx.log`.